crates/cloud/src/client.rs

```diff
@@ -273,6 +273,19 @@ impl Client {
             .await
             .map_err(format_response_error)
     }
+
+    pub async fn list_revisions_next(
+        &self,
+        previous: &RevisionItemPage,
+    ) -> anyhow::Result<RevisionItemPage> {
+        api_revisions_get(
+            &self.configuration,
+            Some(previous.page_index + 1),
+            Some(previous.page_size),
```

Comment:
one more small suggestion, do you think we should pass `None` as `page_size` and let server use  the default `page_size`

Reply:
We could, but doing it this way completely decouples the "next" function from how it got started.  For example, suppose we later modified the starter function to allow the caller to control the page size - we'd then need to modify this function as well (and might not realise until someone got burned by it!).  Doing it this way costs us nothing and breaks a possible future coupling.